content/en/docs/ops/best-practices/security/index.md

```diff
@@ -675,4 +675,74 @@ While most cloud providers support this feature now, many local development tool
 
 By default, Istio (and Envoy) have no limit on the number of downstream connections. This can be exploited by a malicious actor (see [security bulletin 2020-007](/news/security/istio-security-2020-007/)). To work around you this, you must configure an appropriate connection limit for your environment.
 
+The `global_downstream_max_connections` value can be set when installing and configuring Istio via Helm or patching the ingress gateway deployment.
+
+### Configure `global_downstream_max_connections` value with Helm
```

Comment:
```suggestion
### Configure `global_downstream_max_connections` value
```

It applies to all install methods

Reply:
addressed with 6a9e46f17e86ed44c154aad63ade29be5d6c38be